ISSUE_TEMPLATE.md

```diff
@@ -5,12 +5,20 @@
 - Primary contact (assignee):
 - Responsible SIGs:
 - Enhancement target (which target equals to which milestone):
-  - Alpha release target (x.y)
-  - Beta release target (x.y)
-  - Stable release target (x.y)
-- Documentation PR: <!-- link to kubernetes/website PR -->
-  - Alpha:
-  - Beta:
-  - Stable:
+  - Alpha release target (x.y):
+  - Beta release target (x.y):
+  - Stable release target (x.y):
+- [ ] Alpha
+  - [ ] KEP (`k/enhancements`) update PR(s):
+  - [ ] Code (`k/k`) update PR(s):
+  - [ ] Docs (`k/website`) update PR(s):
+- [ ] Beta
+  - [ ] KEP (`k/enhancements`) update PR(s):
+  - [ ] Code (`k/k`) update PR(s):
+  - [ ] Docs (`k/website`) update(s):
+- [ ] Stable
+  - [ ] KEP (`k/enhancements`) update PR(s):
+  - [ ] Code (`k/k`) update PR(s):
+  - [ ] Docs (`k/website`) update(s):
```

Comment:
```suggestion
<!-- Uncomment these as you prepare the enhancement for the next stage 
- [ ] Beta
  - [ ] KEP (`k/enhancements`) update PR(s):
  - [ ] Code (`k/k`) update PR(s):
  - [ ] Docs (`k/website`) update(s):
- [ ] Stable
  - [ ] KEP (`k/enhancements`) update PR(s):
  - [ ] Code (`k/k`) update PR(s):
  - [ ] Docs (`k/website`) update(s):
-->
```

to @kikisdeliveryservice's [suggestion](https://github.com/kubernetes/enhancements/pull/2108/files#r509484503), you could hide these in a comment.

Reply:
Ah I like this!

Reply:
it's been updated! thanks for the suggestion!